runelite-client/src/main/java/net/runelite/client/plugins/devtools/BorderOverlay.java

```diff
@@ -0,0 +1,195 @@
+package net.runelite.client.plugins.devtools;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.geom.GeneralPath;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.Perspective;
+import net.runelite.api.Point;
+import net.runelite.api.coords.LocalPoint;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+
+public class BorderOverlay extends Overlay
+{
+	private static final Color GREEN = Color.GREEN;
+	private static final Color BLUE = Color.BLUE;
+
+	private final Client client;
+	private final DevToolsPlugin plugin;
+
+	@Inject
+	public BorderOverlay(Client client, DevToolsPlugin plugin)
+	{
+		setPosition(OverlayPosition.DYNAMIC);
+		setLayer(OverlayLayer.ABOVE_SCENE);
+		this.client = client;
+		this.plugin = plugin;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (plugin.isToggleChunkBorders())
+		{
+			renderChunkBorders(graphics);
+		}
+
+		if (plugin.isToggleMapSquares())
+		{
+			renderMapSquares(graphics);
+		}
+
+		return null;
+	}
+
+	private void renderChunkBorders(Graphics2D graphics)
+	{
+		final int LOCAL_TILE_SIZE = Perspective.LOCAL_TILE_SIZE;
+		final int CHUNK_SIZE = 8;
+		final int CULL_RANGE = 16;
+		final int STROKE_WIDTH = 4;
```

Comment:
These fields should be constants at top of the file, final fields in instance methods != constants, so UPPER_CASE cannot be used here.